sprockets/mixins/metrics/statsd.py

```diff
@@ -128,10 +160,17 @@ def send(self, path, value, metric_type):
         """
         msg = '{0}:{1}|{2}'.format(
             self._build_path(path, metric_type), value, metric_type)
+
+        encoding = 'utf-8'
+        #encoding = 'ascii'
```

Comment:
Don't leave commented code here. Also, why are you `utf-8` encoding this and having a variable assignment for this on every invocation? I don't believe we support UTF-8 in graphite metric names.